.envrc

```diff
@@ -0,0 +1 @@
+use flake
```

Comment:
This file isn't mentioned in your PR comment.

Reply:
Good catch, I forgot to mention this. It's a small bit of config for [direnv](https://direnv.net/) so that when a Direnv+Nix user changes into the directory the flake dev environment is activated, and when you leave the directory it's unloaded automatically. 

Would you prefer I drop this file or is it OK if documented along with the rest?

Reply:
I updated the PR desc and added a note about direnv support in the new README content about the flake support.